spras/prm.py

```diff
@@ -47,3 +83,35 @@ def validate_required_inputs(cls, filename_map: dict[str, str]):
         for input_type in cls.required_inputs:
             if input_type not in filename_map:
                 raise ValueError("{input_type} filename is missing")
+
+    @classmethod
+    def validate_required_run_args(cls, inputs: dict[str, str | os.PathLike], relax: Optional[list[str]] = None):
+        """
+        Validates the `inputs` parameter for `PRM#run`.
+
+        @param inputs: See `PRM#run`.
+        @param relax: List of inputs that aren't required: if they are specified, they should be valid path
+        """
+        if not relax: relax = []
```

Comment:
What is the relax parameter supposed to do? I'm confused when someone would have a list of inputs that aren't required; are these all optional parameters?

Reply:
[We use this in OmicsIntegrator1](https://github.com/Reed-CompBio/spras/pull/329/files#diff-eed6dce68f02179a25278ef3edc5cd193fcbae298acc1f5969cd5e69dd408e47R161) for the optional dummy node file.